library/crux

```diff
@@ -1,4 +1,16 @@
-# maintainer: James Mills <prologic@shortcircuit.net.au> (@prologic)
+Maintainers: James Mills <prologic@shortcircuit.net.au> (@prologic)
+GitRepo: https://github.com/cruxlinux/docker-crux
 
-latest: git://github.com/prologic/docker-crux@c614d61c53c05c02a43a34187fa1370db2c61524
-3.1: git://github.com/prologic/docker-crux@c614d61c53c05c02a43a34187fa1370db2c61524
+Tags: 3.4, latest
+Architectures: aarch64, amd64
```

Comment:
In my view, this should be ```arm64v8``` instead of ```aarch64```.
I am still doubtful that Docker-Hub supports aarch64 naming or not.

```suggestion
Architectures: arm64v8, amd64
```
Apart from this, it is better to make ```Architectures``` label as global as it is pointing to the same set of architectures in both tags.

Reply:
Thank you for your comments @odidev. 

Ah yes.  Docker Hub is awkward in that regard.

I did swap out x86_64 for amd64, but I forgot about aarch64 for arm64v8.

Will change this.

With regard to the global architectures tag.  This is file is automatically generated by a script.  In order to future-proof the addition of any new architectures in later versions, I think it's best that we separate them.  For instance, a previous version 3.1 does not support AArch64.  I'll push that version to make this clear.